### ISSUE TITLE
handle pipelines with wkt2_2019 syntax

### ISSUE THREAD

**Issue by msmitherdc** [PDAL/PDAL#3423]
pdal 2.2 errors with wkt2_2019 projection syntax in pipeline

```
(pdal pipeline filters.reprojection Error) GDAL failure (1) PROJ: proj_as_wkt: DatumEnsemble can only be exported to WKT2:2019
(pdal pipeline Error) GDAL failure (1) PROJ: proj_as_wkt: DatumEnsemble can only be exported to WKT2:2019
(pdal pipeline Error) GDAL failure (1) PROJ: proj_create: unrecognized format / unknown name
(pdal pipeline Error) GDAL failure (6) Cannot find coordinate operations from `' to `PROJCRS["WGS 84 / UTM zone 11N",BASEGEOGCRS["WGS 84",ENSEMBLE["World Geodetic System 1984 ensemble",MEMBER["World Geodetic System 1984 (Transit)"],MEMBER["World Geodetic System 1984 (G730)"],MEMBER["World Geodetic System 1984 (G873)"],MEMBER["World Geodetic System 1984 (G1150)"],MEMBER["World Geodetic System 1984 (G1674)"],MEMBER["World Geodetic System 1984 (G1762)"],ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1]],ENSEMBLEACCURACY[2.0]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433]],ID["EPSG",4326]],CONVERSION["UTM zone 11N",METHOD["Transverse Mercator",ID["EPSG",9807]],PARAMETER["Latitude of natural origin",0,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8801]],PARAMETER["Longitude of natural origin",-117,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8802]],PARAMETER["Scale factor at natural origin",0.9996,SCALEUNIT["unity",1],ID["EPSG",8805]],PARAMETER["False easting",500000,LENGTHUNIT["metre",1],ID["EPSG",8806]],PARAMETER["False northing",0,LENGTHUNIT["metre",1],ID["EPSG",8807]]],CS[Cartesian,2],AXIS["(E)",east,ORDER[1],LENGTHUNIT["metre",1]],AXIS["(N)",north,ORDER[2],LENGTHUNIT["metre",1]],USAGE[SCOPE["Engineering survey, topographic mapping."],AREA["Between 120°W and 114°W, northern hemisphere between equator and 84°N, onshore and offshore. Canada - Alberta; British Columbia (BC); Northwest Territories (NWT); Nunavut. Mexico. United States (USA)."],BBOX[0,-120,84,-114]],ID["EPSG",32611]]'
```

**Comment by msmitherdc** [comment]
```
{"pipeline": [{"filename": "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_OR_HarneyCounty_2018_LAS_2019/ept.json", "origin": "USGS_LPC_OR_HarneyCounty_2018_Harney_0650_LAS_2019.laz", "bounds": "( [-13247583.868132794, -13245692.387587558], [5411951.169931216, 5412841.896444175] )", "type": "readers.ept"}, {"out_srs": "PROJCS[\"WGS 84 / UTM zone 11N\",GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]],PROJECTION[\"Transverse_Mercator\"],PARAMETER[\"latitude_of_origin\",0],PARAMETER[\"central_meridian\",-117],PARAMETER[\"scale_factor\",0.9996],PARAMETER[\"false_easting\",500000],PARAMETER[\"false_northing\",0],UNIT[\"metre\",1,AUTHORITY[\"EPSG\",\"9001\"]],AXIS[\"Easting\",EAST],AXIS[\"Northing\",NORTH],AUTHORITY[\"EPSG\",\"32611\"]]", "type": "filters.reprojection"}, {"polygon": "POLYGON ((338338.591249167 4835945.48467031,338310.608907261 4835324.35024311,339681.140654654 4835304.29556424,339668.216948229 4835935.00711403,338338.591249167 4835945.48467031))", "outside": false, "type": "filters.crop"}, {"type": "filters.stats"}, {"filename": "/u02/tmp_exports/2/1441/2039/USGS_LPC_OR_HarneyCounty_2018_Harney_0650_LAS_2019.laz", "compression": true, "minor_version": 2, "scale_x": 0.001, "scale_y": 0.001, "scale_z": 1e-05, "offset_x": "auto", "offset_y": "auto", "offset_z": "auto", "forward": "all, vlr", "type": "writers.las"}]}

```

**Comment by msmitherdc** [comment]
```
{"pipeline": [{"filename": "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_OR_HarneyCounty_2018_LAS_2019/ept.json", "origin": "USGS_LPC_OR_HarneyCounty_2018_Harney_0650_LAS_2019.laz", "bounds": "( [-13247583.868132794, -13245692.387587558], [5411951.169931216, 5412841.896444175] )", "type": "readers.ept"}, {"out_srs": "PROJCRS[\"WGS 84 / UTM zone 11N\",BASEGEOGCRS[\"WGS 84\",ENSEMBLE[\"World Geodetic System 1984 ensemble\",MEMBER[\"World Geodetic System 1984 (Transit)\"],MEMBER[\"World Geodetic System 1984 (G730)\"],MEMBER[\"World Geodetic System 1984 (G873)\"],MEMBER[\"World Geodetic System 1984 (G1150)\"],MEMBER[\"World Geodetic System 1984 (G1674)\"],MEMBER[\"World Geodetic System 1984 (G1762)\"],ELLIPSOID[\"WGS 84\",6378137,298.257223563,LENGTHUNIT[\"metre\",1]],ENSEMBLEACCURACY[2.0]],PRIMEM[\"Greenwich\",0,ANGLEUNIT[\"degree\",0.0174532925199433]],ID[\"EPSG\",4326]],CONVERSION[\"UTM zone 11N\",METHOD[\"Transverse Mercator\",ID[\"EPSG\",9807]],PARAMETER[\"Latitude of natural origin\",0,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8801]],PARAMETER[\"Longitude of natural origin\",-117,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8802]],PARAMETER[\"Scale factor at natural origin\",0.9996,SCALEUNIT[\"unity\",1],ID[\"EPSG\",8805]],PARAMETER[\"False easting\",500000,LENGTHUNIT[\"metre\",1],ID[\"EPSG\",8806]],PARAMETER[\"False northing\",0,LENGTHUNIT[\"metre\",1],ID[\"EPSG\",8807]]],CS[Cartesian,2],AXIS[\"(E)\",east,ORDER[1],LENGTHUNIT[\"metre\",1]],AXIS[\"(N)\",north,ORDER[2],LENGTHUNIT[\"metre\",1]],USAGE[SCOPE[\"Engineering survey, topographic mapping.\"],AREA[\"Between 120\u00b0W and 114\u00b0W, northern hemisphere between equator and 84\u00b0N, onshore and offshore. Canada - Alberta; British Columbia (BC); Northwest Territories (NWT); Nunavut. Mexico. United States (USA).\"],BBOX[0,-120,84,-114]],ID[\"EPSG\",32611]]", "type": "filters.reprojection"}, {"polygon": "POLYGON ((338338.591249167 4835945.48467031,338310.608907261 4835324.35024311,339681.140654654 4835304.29556424,339668.216948229 4835935.00711403,338338.591249167 4835945.48467031))", "outside": false, "type": "filters.crop"}, {"type": "filters.stats"}, {"filename": "/u02/tmp_exports/2/1441/2041/USGS_LPC_OR_HarneyCounty_2018_Harney_0650_LAS_2019.laz", "compression": true, "minor_version": 2, "scale_x": 0.001, "scale_y": 0.001, "scale_z": 1e-05, "offset_x": "auto", "offset_y": "auto", "offset_z": "auto", "forward": "all, vlr", "type": "writers.las"}]}
```

**Comment by abellgithub** [comment]
This works with PROJ 8. I get the following SRS in the output LAS file:

> "PROJCS[\"WGS 84 / UTM zone 11N\",GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]],PROJECTION[\"Transverse_Mercator\"],PARAMETER[\"latitude_of_origin\",0],PARAMETER[\"central_meridian\",-117],PARAMETER[\"scale_factor\",0.9996],PARAMETER[\"false_easting\",500000],PARAMETER[\"false_northing\",0],UNIT[\"metre\",1,AUTHORITY[\"EPSG\",\"9001\"]],AXIS[\"Easting\",EAST],AXIS[\"Northing\",NORTH],AUTHORITY[\"EPSG\",\"32611\"]]"

Not sure if this is acceptable...

**Comment by abellgithub** [comment]
Fixed by #3427
